### PR TITLE
Update localization.py

### PR DESCRIPTION
**Pull request by leonardo834**

### CODE REVIEW

File: app/localization.py [hunk 1]
```diff
@@ -90,5 +90,49 @@
 Обрати внимание, что ты можешь использовать меня в любом чате через inline-режим!
 """
     }
```
Comment (kozalosev) [inline app/localization.py]:
```suggestion
    },
    
```

File: app/localization.py [hunk 2]
```diff
@@ -90,5 +90,49 @@
...
+    'pt-BR': {
```
Comment (kozalosev) [inline app/localization.py]: How much is PT-BR different from PT-PT? Maybe, is it better to use more general `'pt'` for the localization?

Comment (leonardo834) [reply app/localization.py]: Let me check if there will be any difference.

Comment (leonardo834) [reply app/localization.py]: There are some differences, but nothing critical. Will update the file.

**Review by kozalosev — APPROVED** [review]
Thank you very much for your contribution!

**Pull request merged**